ui/i18n/src/main/res/values/strings.xml

```diff
@@ -377,6 +377,7 @@
     <string name="date">Date</string>
     <string name="duration">Duration</string>
     <string name="episode_title">Episode title</string>
+    <string name="user_default">User Default</string>
```

Comment:
We call this "Global default" everywhere else, there should be an existing string somewhere, I think